README.md

```diff
@@ -28,7 +28,10 @@ Deze compliancy-afspraken waarborgen dat een hierboven, als eerste, genoemde bel
 
 
 ## Toelichting op bestandsuitwisseling
-De API kan gebruikt worden om de desbetreffende gegevens in json-formaat als bestand op te vragen. Zonder selectie resulteert dat in alle regelteksten en bijbehorende annotaties (locaties, 'juridische regels voor iedereen', activiteiten e.d. zoals beheerd in de plansoftware) of bijvoorbeeld alle locaties. Selectie van de te verkrijgen gegevens is mogelijk:  
+De API kan gebruikt worden om de desbetreffende gegevens in json-formaat als bestand op te vragen.
```

Comment:
Deze API kan niet voor bestandsuitwisseling worden gebruikt. Je kan het JASON-schema dat in deze API is beschreven gebruiken om een JSON bestand uit te wisselen. Dat is echt iets anders. Met deze API kan je nergens een compleet bestand opvragen of sturen.

Reply:
Ik heb de gehele eerste zin van deze paragraaf verwijderd.